apps/docs/src/content/docs/explainers/task-modifiers.mdx

```diff
@@ -3,4 +3,199 @@ title: Task modifiers
 description: How task modifiers work
 ---
 
-This is a guide about how to set up the Async Transform!
+import Timeline from '../TimelineExplainer.svelte';
+
+In our [Getting started](/getting-started/usage#task-modifiers) you've seen that you can specify a
+modifier on the task, either with the dot notation or the options notation. The reason you have this
+ability is so that you can specify what should happen when multiple, concurrent, task instance are
+running.
+
+In this explainer we will guide you through how each of our modifiers work with examples of how they
+behave and hopefully make it clearer to understand when to use one or the other.
+
+## Default
+
+By default, every task simply runs as soon as you perform it. This is basically like a function call
+with the added benefit of having derived state and cancellability. You might've seen in our home
+page this simple demonstration; but if you didn't, allow me to introduce to you the friend that will
+guide us through discovering the task modifiers: the Timeline demo!
+
+Take your time to familiarize yourself with it: you can perform a new task by clicking on the
+`perform` button and reset the demo with the `clear timeline` button, the moment you perform a task
+the timeline will start and it will show you the state of the task. Each task will run for two
+seconds and you can either click on it individually to cancel it or click on the `cancelAll` button
+to nuke them all!
+
+<Timeline show_max={false} client:load />
```

Comment:
I am not sure if the timeline behavior after all the tasks are finished or cancel is expected? It feels a bit weird to me.

https://github.com/user-attachments/assets/c960088e-5da6-4a39-a5ad-3f7eee7a36a7

Reply:
Yeah for the moment is expected.